<launch>
    <arg name="device_id" default="imu"/>
    <arg name="publish_rate" default="1"/>
    <arg name="updates_per_publish" default="1000"/>

    <node name="imu" pkg="peripherals" type="imu" respawn="true" respawn_delay="1" >
        <param name="device_id" value="$(arg device_id)"/>
        <param name="publish_rate" value="$(arg publish_rate)"/>
        <param name="updates_per_publish" value="$(arg updates_per_publish)"/>
    </node>
</launch>
